lighthouse-core/gather/gatherers/script-elements.js

```diff
@@ -120,10 +122,10 @@ class ScriptElements extends Gatherer {
           source: 'network',
           requestId: record.requestId,
           content,
+          node,
```

Comment:
in this case we have a script (somehow), but no matching DOM node.

same story as the linkElements one.